utils/provider.py

```diff
@@ -35,6 +36,14 @@ async def fetch_remote_data(self):
         try:
             while True:
                 try:
+                    # global update
+                    print("Fetching data from", ADSBLOL_HUB_HTTP)
+                    async with self.client_session.get(
+                        f"{ADSBLOL_HUB_HTTP}/data/stats.json"
```

Comment:
could you make this a {ADSBLOL_STATS_URL} instead? I realise I am not sure if /data/stats.json is right and seems like something that might change. Default to `http://hub-readsb:150/stats.json`